.structure-config

```diff
@@ -1 +1 @@
-INFRASTRUCTURE_FILES='.structure-config .packit.yml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
+INFRASTRUCTURE_FILES='.structure-config .packit.yml .readthedocs.yaml .pep8speaks.yml .codecov.yml .coveragerc .github/ dockerfile/ scripts/testing/ scripts/githooks/'
```

Comment:
To resolve failing infra check please add also `docs/requirements.txt`. It's definitely an infra file.

Reply:
Fixed.